### ISSUE TITLE
NullPointerException when Flexbox doesn't have any children

### ISSUE THREAD

**Issue by krobert** [google/flexbox-layout#63]
Hi, i'm receiving this crash when i add a flexbox to my layout without any children, i'm using version 0.2.0. Temporary fixed it by adding a 0dp size Space children.
```
 java.lang.NullPointerException: Attempt to invoke virtual method 'int android.util.SparseIntArray.size()' on a null object reference
                                                                                       at com.google.android.flexbox.FlexboxLayout.isOrderChangedFromLastMeasurement(FlexboxLayout.java:403)
                                                                                       at com.google.android.flexbox.FlexboxLayout.onMeasure(FlexboxLayout.java:259)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
                                                                                       at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
                                                                                       at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:715)
                                                                                       at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:461)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
                                                                                       at android.support.v7.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:135)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
                                                                                       at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
                                                                                       at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
                                                                                       at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
                                                                                       at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
                                                                                       at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
                                                                                       at com.android.internal.policy.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2643)
                                                                                       at android.view.View.measure(View.java:18794)
                                                                                       at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2100)
                                                                                       at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1216)
                                                                                       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1452)
                                                                                       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
                                                                                       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
                                                                                       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
                                                                                       at android.view.Choreographer.doCallbacks(Choreographer.java:670)
                                                                                       at android.view.Choreographer.doFrame(Choreographer.java:606)
                                                                                       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
                                                                                       at android.os.Handler.handleCallback(Handler.java:739)
                                                                                       at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                                       at android.os.Looper.loop(Looper.java:148)
                                                                                       at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by thagikura** [comment]
Hi,

Thanks for the report.
Oops, sorry. I'm going to fix it soon.

**Comment by thagikura** [comment]
Fixed by #64 and the new version 0.2.1 is now available. Confirmed it works with empty children.
Sorry about that.

**Comment by krobert** [comment]
Cool thanks for the quick fix.
